package.json

```diff
@@ -9,6 +9,16 @@
     "build": "graph build",
     "create-local": "graph create dynamic-amm/dynamic-amm --node http://127.0.0.1:8020",
     "deploy-local": "graph deploy dynamic-amm/dynamic-amm --debug --ipfs http://localhost:5001 --node http://127.0.0.1:8020",
+    "create-kyber-avalanche": "graph create kybernetwork/kyberswap-avalanche --node https://avalanche-graph.kyberengineering.io/admin",
```

Comment:
now we have the same structure for all chains, we can use something similar to the prepare script, use `NETWORK` param so we don't need to have a lot of scripts like this

Reply:
I've updated. please check again